sdf.md

```diff
@@ -215,40 +219,71 @@ implementations to SDF base.
 
 ## Terminology and Conventions
 
+### Conceptual Terms
+{:unnumbered}
+
 Thing:
 : A physical item that is also available for interaction over a network.
 
-Grouping:
-: An sdfThing or sdfObject, i.e., (directly or indirectly) a combination of Affordances.
+Element:
+: A part or an aspect of something abstract; i.e., the term is used
+  here in its usual English definition.
+  (Exceptionally, in {{jso-inspired}}, used explicitly for its meaning in the JSON
+  ecosystem, i.e., the elements of JSON arrays.)
 
 Affordance:
-: An element of an interface offered for interaction, for which
-  information is available (directly or indirectly) that indicates how
-  it can or should be used.
-  The term is used here for the digital (network-directed) interfaces
-  of a Thing only; the Thing
-  might also have physical affordances such as buttons, dials, and
-  displays.
+: An element of an interface offered for interaction.
+  Such an element becomes an Affordance when information is available
+  (directly or indirectly) that indicates how it can or should be
+  used.
+  In the present document, the term is used for the digital
+  (network-directed) interfaces of a Thing only; as it is a physical
+  object as well, the Thing might also have physical affordances such
+  as buttons, dials, and displays.
 
-Quality:
-: A metadata item in a definition or declaration which says something
-  about that definition or declaration.  A quality is represented in
-  SDF as an entry in a JSON map (JSON object) that serves as a definition
-  or declaration.
+Property:
+: An affordance that can potentially be used to read, write, and/or
+  observe state (current/stored information) on a Grouping.\\
+  (Note that other environments often use the term "property" for a
+  JSON concept that we call "entry" [see below].
+  In this document, the term Property is specifically reserved for
+  affordances, even if the map key `"properties"` might be imported with
+  the JSON-level semantics from a data definition language.)
+
+Action:
+: An affordance that can potentially be used to perform a named operation on a Grouping.
```

Comment:
Hm, one thing I've noticed is that sometimes terms that are formally defined here are capitalized (e.g., in this line "Grouping") and sometimes not (e.g., "affordance"). Would it make sense to capitalize all the terms defined here to make it more obvious that they are part of the terminology?